Library/Homebrew/formula.rb

```diff
@@ -1622,6 +1622,19 @@ def std_meson_args
     ["--prefix=#{prefix}", "--libdir=#{lib}", "--buildtype=release", "--wrap-mode=nofallback"]
   end
 
+  # Standard parameters for pip builds.
+  sig {
+    params(prefix:          T.nilable(T.any(String, Pathname)),
```

Comment:
```suggestion
    params(prefix:          T.any(String, Pathname, FalseClass),
```

might be nicer as I think `prefix: false` reads nicer than `prefix: nil`.

Reply:
agreed, thanks!